akka-stream/src/main/scala/akka/stream/impl/fusing/Ops.scala

```diff
@@ -15,13 +16,13 @@ import akka.stream.Attributes.{ InputBuffer, LogLevels }
 import akka.stream.OverflowStrategies._
 import akka.stream.impl.Stages.DefaultAttributes
 import akka.stream.impl.fusing.GraphStages.SimpleLinearGraphStage
-import akka.stream.impl.{ ReactiveStreamsCompliance, Buffer => BufferImpl, ContextPropagation }
+import akka.stream.impl.{ ContextPropagation, ReactiveStreamsCompliance, Buffer => BufferImpl }
```

Comment:
My IDE reorganized imports, it does look better, but not sure the ordering is right here.
I can reserve the changes here.

Reply:
Not important